add rv3 to changelog (#263)
CQCL/tket: pytket/docs/changelog.rst

* The ``QubitPauliOperator.from_OpenFermion`` and
  ``QubitPauliOperator.to_OpenFermion`` methods are removed.

Major new features:

* New methods for mapping logical to physical circuits for some ``Architecture``.

>   The new method will use a list of user given method, each of them suitable only 

Reviewer: ```suggestion
  The new method will use a list of user-given methods, each of them suitable only 
```

>   for a specific set of sub circuits. Users can add there own methods if they want to.

Reviewer: ```suggestion
  for a specific set of subcircuits. Users can add their own methods if they want to.
```

  Take a look on the `mapping_example <https://github.com/CQCL/pytket/blob/main/examples/mapping_example.ipynb>`_

>   notebook to find out more. All compiler passes in pytket are updated to use the new method.

Reviewer: Let's leave out the reference to the notebook. (It isn't live yet,)

Author: I have removed it

Author: see [7114822](https://github.com/CQCL/tket/pull/263/commits/7114822c829f9819dd282c45b3910a86a297fc7d)

>   notebook to find out more. All compiler passes in pytket are updated to use the new method.

Reviewer: ```suggestion
  notebook to find out more. All compiler passes in pytket are updated to use the new methods.
```

  The methods already given by pytket are ``LexiRouteRoutingMethod``,
  ``LexiLabellingMethod``, ``MultiGateReorderRoutingMethod``,

>   ``BoxDecompositionRoutingMethod``, ``AASRouteRoutingMethod`` and ``AASLabellingMethod``.

Reviewer: Leave out `BoxDecompositionMethod` (not merged yet).

Author: I have removed it

Author: see [7114822](https://github.com/CQCL/tket/pull/263/commits/7114822c829f9819dd282c45b3910a86a297fc7d)
